spec/features/sign_in_to_Chitter_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'User sign in' do
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)